app/src/main/java/com/twilio/video/app/ui/room/RoomViewModel.kt

```diff
@@ -145,18 +148,10 @@ class RoomViewModel(
     }
 
     private fun subscribeToRoomChannel() {
```

Comment:
Do you want to refactor the method to indicate that Channel is not used now?

Reply:
Good catch! Renaming now...